Editor/API/Util/MiscHelpers.cs

```diff
@@ -12,24 +12,14 @@ namespace nadena.dev.ndmf.util
     public static class MiscHelpers
     {
         [CanBeNull]
+        [Obsolete("Please use RuntimeUtil.RelativePath() instead.")]
         public static string RelativePath(GameObject root, GameObject child)
         {
-            if (root == child) return "";
-
-            List<string> pathSegments = new List<string>();
-            while (child != root && child != null)
-            {
-                pathSegments.Add(child.name);
-                child = child.transform.parent?.gameObject;
-            }
-
-            if (child == null && root != null) return null;
-
-            pathSegments.Reverse();
-            return String.Join("/", pathSegments);
+            return RuntimeUtil.RelativePath(root, child);
         }
 
         [CanBeNull]
+        [Obsolete("Please use RuntimeUtil.AvatarRootPath() instead.")]
         public static string AvatarRootPath(this GameObject child)
```

Comment:
RuntimeUtil isn't an extension method, I'd prefer to keep the convenience around...

Reply:
Oh, sorry.